content/modules/ROOT/pages/index.adoc

```diff
@@ -605,20 +603,20 @@ Training using the newly generated data is a time and resource intensive task. D
 
 At this point, we are ready to serve our model and test it out. Keep in mind that we only generated 5 additional data points, so the results will vary, and we can’t guarantee the skill we are looking to add was generated with such a small sample size. For that reason, we are going to do a bit of a cooking-show style serving where we are going to serve a model using the exact steps we did in this workshop but with an instruction size of 100 instead of 5. This is simply due to the time constraints; otherwise, you would be sitting here for a few hours, and that isn’t going to make you happy. ;) 
 
-NOTE: Make sure to stop the previous `ilab serve` command that may still have running in the other terminal tab.
+NOTE: Make sure to stop the previous `ilab model serve` command that may still have running in the other terminal tab.
 
 [source,sh,role=execute,subs=attributes+]
 ----
-ilab serve --model-path models/ggml-ilab-pretrained-Q4_K_M.gguf
+ilab model serve --model-path models/ggml-ilab-pretrained-Q4_K_M.gguf
 ----
 
-NOTE: If you get an error message about not being able to bind to the address, simply kbd:[CTRL+C]  the `ilab serve` command that may still have running in your other terminal tab.
+NOTE: If you get an error message about not being able to bind to the address, simply kbd:[CTRL+C]  the `ilab model serve` command that may still have running in your other terminal tab. 
 
-Start up another chat session with it:
+Start up another chat session with it. You will add the kbd:[--greedy-mode] flag to minimize any potential response randomness or variation in the generated response:
```

Comment:
Perhaps we explain that it lowers the model temperature to 0 to help with deterministic-ness?

Reply:
I left that specificity out because Grant said it sets to 0 "supposedly". So maybe when we confirm that functionality we can be more direct.